src/components/YearbookSales/SalesBoxChart/SalesBoxChart.js

```diff
@@ -0,0 +1,33 @@
+import "./salesBoxChart.less"
+import * as React from "react";
+import sizeMe from 'react-sizeme';
+import Bubble from "../../Common/Bubble";
+
+class SalesBoxChart extends React.Component {
+    render() {
+        const {max, campus, online, size} = this.props;
+        const width = size.width - 26;
```

Comment:
what is 26? I think this should be constants, declared above component or if you will be use this constants in another file, constans should be declared in external file. 
for example:
`const MARGIN_WIDTH = 26`